onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -4312,6 +4312,102 @@ TEST_F(GraphTransformationTests, BitmaskDropoutFusionTest) {
   TestBitmaskDropoutFusion(MODEL_FOLDER "fusion/bitmask_bias_dropout_fusion_residual.onnx", true, *logger_, 0, 0, 0, 0,
                            1, 0, 1);
 }
+
+/*
+This test build a graph like:
+	           input0  input1
+	                \ /
+                    Add
+	-----------------|
+	|                |
+  |             Shape
+	|             /   \
+  |        Gather0  Gather1
+  |           /       \
+  |     Unsqueeze0  Unsqueeze1
+	|           \        /
+	|     	     \      /
+	|         ConcatTraining
+	 \	       /
+	  \	      /
+	    Reshape
+
+  After fusion, the graph become:
+             input0  input1
+	                \ /
```

Comment:
there should be the connection between input 1 and constant initilaizer?

Reply:
Checked the code, I think there is no connection between input and constant initializer.

Reply:
OK I mean the comment did not look right. But now it is good.